Description: Disable failing unittests.
Forwarded: no

Some of those tests are known to be unreliable. We disable
them. Upstream is working on that.

diff --git a/qa/tests/control_test.py b/qa/tests/control_test.py
index ae03bf01b763..46ed92330f99 100755
--- a/qa/tests/control_test.py
+++ b/qa/tests/control_test.py
@@ -68,6 +68,7 @@ class TestControl (unittest.TestCase):
 	def test_one_newline (self):
 		self.validate('x\n','x')
 
+	@unittest.skip
 	def test_two_newline (self):
 		self.validate('-\nx\n','x')
 
